socotratools/socotratools/client.py

```diff
@@ -522,47 +533,54 @@ def get_endorsements_from_policy(self, policy_locator):
         return self.__get("/policies/" + policy_locator +
                           "/endorsements")
 
-    def create_renewal(self, policy_locator, end_timestamp=None,
-                       field_values={}, add_field_groups=[],
-                       update_field_groups=[], remove_field_groups=[],
-                       add_exposures=[], update_exposures=[],
-                       end_exposures=[], action=None):
-
-        data = {'renewalEndTimestamp': end_timestamp,
+    def create_renewal(self, policy_locator,
+                       field_values=None,
+                       add_field_groups=None,
+                       remove_field_groups=None,
+                       update_field_groups=None,
+                       add_exposures=None,
+                       end_exposures=None,
+                       update_exposures=None,
+                       # above: optional
+                       end_timestamp=None,
```

Comment:
Unlike endorsements, renewals require a new policy (effective contract) end timestamp. In the renewal update, the effective contract is optional because one was provided upon creation.

Note: There is probably some validation we can do to clarify usage. Making this a keyword argument rather than positional doesn't change the fact that it's required to actually create the renewal. (kind of like how endorsements require a named configuration)